app/client/components/identity/ToggleStyles.tsx

```diff
@@ -0,0 +1,137 @@
+import { css } from "@emotion/core";
+import { textSans } from "@guardian/src-foundations/typography"; // TODO v4 -> 2 change
+import { neutral, success } from "@guardian/src-foundations/palette"; // TODO v4 -> 2 change
+
+/**
+ *  TODO: this toggle stylesheet is copied from Source (Design System)
+ *  It should be updated to the Source version once MMA migration to Emotion 11 and Source v3+ is complete
+ */
```

Comment:
Should be able to use the Source toggle switch once PR #770 is merged.  Styles and component copied over with minor change - Addition of `labelId` for testing purposes.